scripts/deploy/execute-deployment-pipeline.mjs

```diff
@@ -0,0 +1,49 @@
+import {execSync} from 'node:child_process';
```

Comment:
This file does not make much sense: There is no Atomic/Bueno/Headless package in this repository to extract the version from. I guess this was copy pasted from ui-kit repo ?